lib/talkable/generators/templates/app/controllers/example_newsletter_signup_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ExampleNewsletterSignupController < ApplicationController
+  def register
+  end
+
+  def submit
+    Talkable.register_event(
+        email: params[:email],
+        event_number: SecureRandom.uuid,
```

Comment:
shouldn't `event_number` be associated with `email` here? The question is whether we need only one event per every signup or many?

Reply:
"#{params[:email}$#{SecureRandom.uuid}" maybe?